### DISCUSSION TITLE
How do I migrate/change database?

### DISCUSSION THREAD

**Discussion by sebzapata** [umami-software/umami#846] [Q&A]
Hi,
I currently have my app set up and deployed via Heroku which was done using the "Running on Heroku" guide in the docs.
I am just about to exceed the 10,000 rows limit of my Postgres DB and would like to change the DB to one with a larger limit from Elephant SQL.
How would I change the database that is written/read from? I assumed I would just need to update the `DATABASE_URL` in the `docker-compose.yml` file, but this doesn't seem to have done what I expected.

Thank you

**Comment by mikecao** [comment]
All configuration is done in the `.env` file.
